README.md

```diff
@@ -71,7 +71,7 @@ Vault.
 If you are using Venafi as a Service, verify the following:
 
 - The Venafi as a Service REST API at [https://api.venafi.cloud](https://api.venafi.cloud/swagger-ui.html)
-is accessible from the systems where Vault will be running.
+  or https://api.eu.venafi.cloud(if you have an EU account) is accessible from the systems where Vault will be running.
```

Comment:
Check which url should be used for EU here as well

Reply:
It's done